ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -686,7 +686,10 @@ ethernet_interfaces:
     l2_mtu: < l2-mtu - if defined this profile should only be used for platforms supporting the "l2 mtu" CLI >
     vlans: "< list of vlans as string >"
     native_vlan: <native vlan number>
-    mode: < access | dot1q-tunnel | trunk >
+    mode: < access | dot1q-tunnel | trunk | trunk-phone >
```

Comment:
This should be `< access | dot1q-tunnel | trunk | "trunk phone" >`

Reply:
Good catch.  This should be resolved.